python: can't open file '/app/sqanti3_lrgasp.challenge3.py': [Errno 2] No such file or directory
